debug/src/SimpleServer.hs

```diff
@@ -23,7 +22,6 @@ import Data.Default.Class
 import Data.IORef
 import Data.Monoid
 import Data.Char (isDigit)
-import Data.X509.Validation
```

Comment:
I removed the `Control.Exception` import. `Data.Monoid` is redundant only in GHC > 7.8. But I shall look into the others again.